test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpLogExporterTests.cs

```diff
@@ -225,28 +225,82 @@ public void CheckToOtlpLogRecordSeverityLevelAndText(LogLevel logLevel)
             }
         }
 
-        [Fact]
-        public void CheckToOtlpLogRecordFormattedMessage()
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void CheckToOtlpLogRecordBodyIsPopulated(bool includeFormattedMessage)
         {
             var logRecords = new List<LogRecord>();
             using var loggerFactory = LoggerFactory.Create(builder =>
             {
                 builder.AddOpenTelemetry(options =>
                 {
                     options.AddInMemoryExporter(logRecords);
-                    options.IncludeFormattedMessage = true;
+                    options.IncludeFormattedMessage = includeFormattedMessage;
+                    options.ParseStateValues = true;
```

Comment:
Do we need a test for when both IncludeFormattedMessage and ParseStateValues is null then Body will be null?

Reply:
yes surely. (There are lot of corner cases as well)
I'll iterate to add more tests.